src/Logary.Facade/Logary.Facade.fsproj

```diff
@@ -30,21 +31,6 @@
     <WarningLevel>3</WarningLevel>
     <DocumentationFile>bin\Release\Logary.Facade.xml</DocumentationFile>
   </PropertyGroup>
-  <ItemGroup>
```

Comment:
Done by VS, I'll remove these when I'm done, otherwise VS will just keep doing it